sql/2020/16_Ecommerce/pagestats_image_bydevice.sql

```diff
@@ -12,7 +12,9 @@ JOIN
 USING (_TABLE_SUFFIX, url),
 UNNEST([10, 25, 50, 75, 90]) AS percentile
 WHERE
-  category = 'Ecommerce'
+  category = 'Ecommerce' AND 
+  (app != 'Cart Functionality' AND 
+   app != 'Google Analytics Enhanced eCommerce')
 GROUP BY
   percentile,
   client
```

Comment:
Always good to add an ORDER BY:

```suggestion
GROUP BY
  percentile,
  client
ORDER BY
  percentile,
  order
```